README.md

```diff
@@ -39,7 +39,20 @@ end
 
 ## Faster cache loading
 
-Add the `msgpack` gem and `require 'msgpack'` to gain ~10-30ms of extra load speed by using msgpack.
+In order to gain ~10-30ms of extra load speed, you can use the msgpack gem:
+
+```ruby
+# Gemfile
+gem 'msgpack',   require: false
+gem 'bootscale', require: false
```

Comment:
well, they'll be required before bundler try to require them, but ok I see the point.